ecosystem/node-checker/src/evaluators/metrics/network/peers.rs

```diff
@@ -0,0 +1,357 @@
+            self.build_evaluation_result(
+                format!(
+                    "There are sufficient {} connections {} the target node",
+                    name, particle
+                ),
+                100,
+                explanation,
+            )
+        } else {
+            self.build_evaluation_result_with_links(
+                format!(
+                    "There are not enough {} connections {} the target node",
+                    name, particle
+                ),
+                50,
+                format!("{} Try setting explicit peers.", explanation),
+                vec![
+                    "https://aptos.dev/nodes/full-node/troubleshooting-fullnode-setup".to_string(),
+                ],
+            )
+        }
+    }
+}
+
+#[async_trait::async_trait]
+impl Evaluator for NetworkMinimumPeersEvaluator {
+    type Input = MetricsEvaluatorInput;
+    type Error = MetricsEvaluatorError;
+
+    async fn evaluate(&self, input: &Self::Input) -> Result<Vec<EvaluationResult>, Self::Error> {
+        let (inbound_connections, outbound_connections) =
+            match self.get_metrics(&input.latest_target_metrics) {
+                Ok((inbound_connections, outbound_connections)) => {
+                    (inbound_connections, outbound_connections)
+                }
+                Err(evaluation_results) => return Ok(evaluation_results),
+            };
+
+        Ok(vec![
+            self.build_evaluation(
+                inbound_connections,
+                self.args.minimum_peers_inbound,
+                &ConnectionType::Inbound,
+            ),
+            self.build_evaluation(
+                outbound_connections,
+                self.args.minimum_peers_outbound,
+                &ConnectionType::Outbound,
+            ),
+        ])
+    }
+
+    fn get_category_name() -> String {
+        CATEGORY.to_string()
+    }
+
+    fn get_evaluator_name() -> String {
+        "minimum_peers".to_string()
+    }
+
+    fn from_evaluator_args(evaluator_args: &EvaluatorArgs) -> Result<Self> {
+        Ok(Self::new(evaluator_args.network_minimum_peers_args.clone()))
+    }
+
+    fn evaluator_type_from_evaluator_args(evaluator_args: &EvaluatorArgs) -> Result<EvaluatorType> {
+        Ok(EvaluatorType::Metrics(Box::new(Self::from_evaluator_args(
+            evaluator_args,
+        )?)))
+    }
+}
+
+impl NetworkPeersEvaluator for NetworkMinimumPeersEvaluator {}
+
+//////////////////////////////////////////////////////////////////////////////
+// Evaluator for number of peers within tolerance.
+//////////////////////////////////////////////////////////////////////////////
+
+#[derive(Clone, Debug, Deserialize, Parser, PoemObject, Serialize)]
+pub struct NetworkPeersWithinToleranceEvaluatorArgs {
+    /// The evaluator will ensure that the inbound connections count is
+    /// within this tolerance of the value retrieved from the baseline.
+    #[clap(long, default_value_t = 10)]
```

Comment:
We'll probably need to tweak this value as time goes on, but `10` seems reasonable for now.